core/lib/zksync_core/src/l1_gas_price/gas_adjuster/mod.rs

```diff
@@ -40,11 +44,23 @@ impl GasAdjuster {
             .await?
             .as_usize()
             .saturating_sub(1);
-        let history = eth_client
+        let base_fee_history = eth_client
             .base_fee_history(current_block, config.max_base_fee_samples, "gas_adjuster")
             .await?;
+        let last_block_blob_base_fee =
+            Self::get_blob_base_fee_history(&eth_client, current_block..current_block).await?;
```

Comment:
current_block..current_block seems strange. please comment.
(as I'd have expected something like current_block-range .. current_block)

Reply:
I've changed to `current_block..=current_block`